src/environment.h

```diff
@@ -48,8 +48,13 @@ class Environment {
 
   void ReadApplicationDefaultCredentials() const;
 
+  // Used to override server address in tests.
+  void SetGceMetadataServerAddress(const std::string& address);
```

Comment:
`SetMetadataServerUrlForTest`, and you can get rid of the comment.

Reply:
Done.